### PR TITLE
Parse pref from type

### PR DESCRIPTION
**Pull request by DeepDiver1975**
refs #173

bildschirmfoto von 2016-03-11 23-53-02

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
@Henni @irgendwie please review - THX

File: js/components/detailsItem/detailsItem_controller.js [hunk 1]
```diff
@@ -15,9 +16,21 @@ app.controller('detailsItemCtrl', ['$templateRequest', 'vCardPropertiesService',
...
+			ctrl.isPreferred = true;
+			array.splice(array.indexOf('PREF'), 1);
+		}
+		ctrl.type = array.join(',');
+		var displayName = array.map(function (element) {
+			return element.charAt(0).toUpperCase() + element.slice(1).toLowerCase();
+		}).join(' ');
+
+		if (!ctrl.availableOptions.some(function(e) { return e.id === ctrl.type; } )) {
+			ctrl.availableOptions = ctrl.availableOptions.concat([{id: ctrl.type, name: displayName}]);
```
Comment (Henni) [inline js/components/detailsItem/detailsItem_controller.js:37]: @DeepDiver1975 please add a few comments here.

**Comment by Henni** [comment]
There isn't a way to set the preferred field yet or did I miss it?

Otherwise looks good.

File: js/components/detailsItem/detailsItem_controller.js (hunk 1 above)
Comment (DeepDiver1975) [inline js/components/detailsItem/detailsItem_controller.js:37]: some comments added

**Comment by DeepDiver1975** [comment]
> There isn't a way to set the preferred field yet or did I miss it?

There is non so far - we can add this in the future

**Comment by irgendwie** [comment]
Rebased...
LGTM :+1:

**Pull request merged**